lib/middleware/assignActiveLicence.js

```diff
@@ -0,0 +1,44 @@
+const { acquisitionCtxClient } = require('@financial-times/kat-client-proxies');
+
+const RE_FTCOM = /^FT\.com\s*/i;
+
+function addLicenceAcquisitionData (licence) {
+	return acquisitionCtxClient.getContexts({'access-licence-id': licence.licenceId})
+		.then(result => {
+			if (Array.isArray(result)) {
```

Comment:
Would this ever not be an array?

Reply:
Looks like the request in kat-client-proxies will return the licence object if no acquisition context is returned

Reply:
Maybe not